examples/python_api_inference/resnet50_inference.ipynb

```diff
@@ -330,12 +286,28 @@
     "cv2.destroyAllWindows()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If script is run on a headless server where .imshow() experiences problems, the following cell can be run to verify functionalty:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "while (cap.isOpened()):\n",
+    "    start = time.perf_counter()\n",
+    "    ret, frame = cap.read()\n",
+    "    if not ret: break\n",
+    "    \n",
+    "    top_prediction = predict_class(frame)\n",
+    "    print(labels[top_prediction])\n",
+    "cap.release()"
+   ]
```

Comment:
Looks good. Only suggestion is that we could maybe show a histogram of predictions once it is finished rather than having it print the prediction for every frame.

Reply:
I like the idea. I will add it and circle back afterwards.